craco.config.js

```diff
@@ -0,0 +1,29 @@
+const { ProvidePlugin } = require('webpack')
+
+module.exports = {
+  webpack: {
+    configure: {
+      resolve: {
+        fallback: {
+          path: require.resolve('path-browserify'),
+          timers: require.resolve('timers-browserify'),
+          fs: require.resolve('browserify-fs'),
+          https: require.resolve('https-browserify'),
+          stream: require.resolve('stream-browserify'),
+          http: require.resolve('stream-http'),
+        },
+      },
+      ignoreWarnings: [/Failed to parse source map/],
```

Comment:
Webpack outputs a bunch of these errors when we use it, but they're all for deep dependencies we will likely never need sourcemaps for.